eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -116,7 +116,38 @@ stages:
                     PackageName: ${{artifact.name}}
                     ServiceName: ${{parameters.ServiceDirectory}}
                     ForRelease: true
-
+          - job: InstallationCheck
```

Comment:
I know the rest of the release pipeline is all dumped out in this one large yaml file, but we could start improving this situation by putting new updates (like this one) into their own job files and including them as a template.